niceday-api/package.json

```diff
@@ -19,9 +19,10 @@
     "@sense-os/goalie-js": "senseobservationsystems/goalie-js#v0.39.2",
     "connect": "^3.2.0",
     "dotenv": "^8.2.0",
+    "isomorphic-fetch": "^3.0.0",
     "js-yaml": "^3.3.0",
     "oas3-tools": "^2.2.3",
-    "isomorphic-fetch": "^3.0.0"
+    "xmlhttprequest": "^1.8.0"
```

Comment:
Did we need xmlhttprequest in the end? No right?

Reply:
Ah yes, good point. Removed it.